Core/src/org/sleuthkit/autopsy/textextractors/TextFileExtractor.java

```diff
@@ -73,7 +76,26 @@ private Reader getReader(Charset encoding) {
     }
 
     @Override
-    public boolean isSupported() {
-        return file.getMIMEType().equals("text/plain");
+    public boolean isSupported() {        
+        // get the MIME type
+        String mimeType = file.getMIMEType();
+        
+        // if it is not present, attempt to use the FileTypeDetector to determine
+        if (StringUtils.isEmpty(mimeType)) {
+            FileTypeDetector fileTypeDetector = null;
+            try {
+                fileTypeDetector = new FileTypeDetector();
+            } catch (FileTypeDetector.FileTypeDetectorInitException ex) {
+                logger.log(Level.WARNING, "Unable to create file type detector for determining MIME type.");
```

Comment:
@rcordovano I'm wondering if we should log this as severe? I know that typically we try to reserve severe for errors that compromise the case, but I think if we are unable to instantiate FileTypeDetector class, then it definitely needs to be investigated.

Reply:
I agree, something is very wrong if we can't do file type detection and it will greatly affect the case.

Reply:
Yep. Everything else looks good.

Reply:
Ok.  This should be resolved now.  Thanks.